stop.sh

```diff
@@ -1,4 +1,61 @@
-#!/usr/bin/env sh
+#!/bin/bash
 
-docker-compose down
-docker-compose -f docker-compose-init.yml down
\ No newline at end of file
+set -o errexit -o pipefail -o noclobber -o nounset
+cd "$(dirname "$0")"
```

Comment:
why these cds?

Reply:
These CDs are so that the working directory is the same as the script's directory. All the relative paths from the script don't rely on where the script is run from.